docs/how-to/use_elastic_store.md

```diff
@@ -0,0 +1,61 @@
+# How-To: Use Elastic to Store Similarity Index
+
+[The similarity index](../concepts/similarity_indexes.md) is a feature provided by db-ally designed to map user input to the closest matching value in a data source. In this guide, we will show you how to execute similarity search by using elastic search.
+In the example the elastic search engine is provided by official docker image.
+
+## Environment setup
+
+* [Download and deploy elastic search docker image](https://www.elastic.co/guide/en/elasticsearch/reference/current/docker.html). Instructions to set up kibana environment are not required.
+
+```commandline
+docker network create elastic
+docker pull docker.elastic.co/elasticsearch/elasticsearch:8.13.4
+docker run --name es01 --net elastic -p 9200:9200 -it -m 1GB docker.elastic.co/elasticsearch/elasticsearch:8.13.4
+# Copy the generated elastic password and enrollment token. These credentials are only shown when you start Elasticsearch for the first time. You can regenerate the credentials using the following commands.
+docker cp es01:/usr/share/elasticsearch/config/certs/http_ca.crt .
+curl --cacert http_ca.crt -u elastic:$ELASTIC_PASSWORD https://localhost:9200
+```
+
+* Store specific (docker container creation printed) output to following variables in the project .venv file:
+  - ELASTIC_STORE_CONNECTION_STRING (e.g ```https://localhost:9200```),
```

Comment:
These envs don't render well in the documentation, they are mixed up with the steps and it's unreadable. Could you list them one by one, like this A, B, C? You don't need to give example values here.